internal/action/command.go

```diff
@@ -485,13 +485,15 @@ func (h *BufPane) HelpCmd(args []string) {
 			return
 		}
 
-		if config.FindRuntimeFile(config.RTHelp, topics[0]) != nil {
-			err := h.openHelp(topics[0], hsplit, forceSplit)
-			if err != nil {
-				InfoBar.Error(err)
+		for _, topic := range topics {
+			if config.FindRuntimeFile(config.RTHelp, topic) != nil {
+				err := h.openHelp(topic, hsplit, forceSplit)
```

Comment:
> If current pane is a help pane and `forceSplit` is false, this opens just the last topic (in this current pane).

So it was before too.

> Maybe let it depend on the multi-open scenario?

The suggestion sounds like a "yes" to my question.
Since using help with more than one topic but opening just one doesn't really make sense it's better to force it in this scenario.